lib/screens/addLocation.js

```diff
@@ -0,0 +1,78 @@
+
+import React, { Component } from 'react';
+import { StyleSheet, View, ScrollView, Button } from 'react-native';
+import Dropdown from './dropdown';
+
+export default class AddLocation extends Component {
+  constructor() {
+    super();
+    this.state = {
+      ViewArray: []
+    }
+    this.Array_Value_Index = 2;
+  }
+  Add_New_View_Function = () => {
+    let New_Added_View_Value = { Array_Value_Index: this.Array_Value_Index }
+    this.setState({ ViewArray: [...this.state.ViewArray, New_Added_View_Value] }, () => {
+      this.Array_Value_Index++;
+    });
+  }
+  render() {
+    let Render_Animated_View = this.state.ViewArray.map((item, key) => {
+      return (
+        <View
+          key={key}
+          style={{ paddingTop: 5, height: 50 }}
+        >
+          <Dropdown placeholder=' End Location' />
```

Comment:
```suggestion
          <Dropdown placeholder='End Location' />
```
Not sure if this space in the string is intentional for formatting?
If it was, we can just add more padding to the view instead

Reply:
Can we do the same for all the other strings in the dropdowns pls